client/src/components/LandingPage/index.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import { makeStyles } from "@material-ui/core";
+import { useHistory, Link } from "react-router-dom";
+
+import FocusLogo from "../../images/focus-logo.svg";
+import styles from "./style.module.scss";
+
+const useStyles = makeStyles({
+    root: {
+        "& .MuiOutlinedInput-root": {
+            borderRadius: 15,
+        },
+        "& .MuiInputLabel-root": {
+            fontSize: "1.25rem",
+            color: "#7D7D7D",
+            fontFamily: "'Ropa Sans', sans-serif",
+        },
+    },
+    buttonRoot: {
+        background: "#30A0F5",
+        borderRadius: 50,
+        padding: "4px 25px",
+        width: "100%",
+    },
+    label: {
+        textTransform: "capitalize",
+        fontFamily: "'Ropa Sans', sans-serif",
+        fontSize: "1.75rem",
+    },
+});
+
+export default function LandingPage() {
+    const classes = useStyles();
+    const history = useHistory();
+
+    return (
+        <div className={styles.page}>
+            <div className={styles.container}>
+                <div className={styles.focusIntro}>
+                    <img className={styles.focusLogo} src={FocusLogo} alt="FOCUS" />
+                    <p>
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc blandit in
```

Comment:
Can we have some real text here: for example here is the description text from the Wiki: **Daily Focus is a dashboard web app that surfaces vital at-a-glance information to enable you to know what you need to know and get on with your day. The dashboard integrates with a number of different data providers and services, and the display of this data can be configured in the form of widgets to best suit your needs.**

Reply:
(or just the first sentence if that description is too long)